src/Features/Core/Portable/EditAndContinue/EditAndContinueCapabilities.cs

```diff
@@ -37,6 +37,11 @@ internal enum EditAndContinueCapabilities
         /// <summary>
         /// Creating a new type definition.
         /// </summary>
-        NewTypeDefinition = 1 << 4
+        NewTypeDefinition = 1 << 4,
+
+        /// <summary>
+        /// Adding, updating and deleting of custom attributes (as distinct from attributes that effect metadata)
+        /// </summary>
+        UpdateCustomAttributes = 1 << 5,
```

Comment:
>UpdateCustomAttributes [](http://example.com/codeflow?start=8&length=22)

Perhaps `ChangeCustomAttributes` would be better since update might mean just updating existing values (this is how we usually use the word in EnC).